src/Propel/Generator/Command/InitCommand.php

```diff
@@ -411,12 +411,8 @@ private function reverseEngineerSchema(OutputInterface $output, array $options)
      */
     protected function createConsoleHelper(InputInterface $input, OutputInterface $output)
     {
-        /* Check if it runs in Symfony3 env — than use QuestionHelper, because DialogHelper is absent */
-        if (class_exists('\Symfony\Component\Console\Helper\QuestionHelper')) {
-            $helper = new ConsoleHelper3($input, $output);
-        } else {
-            $helper = new ConsoleHelper($input, $output);
-        }
+        $helper = new ConsoleHelper3($input, $output);
```

Comment:
Should I go further and remove `Propel\Generator\Command\Helper\ConsoleHelperInterface`.